### DISCUSSION TITLE
How to rotate/Orbit in Y axis Mobile

### DISCUSSION THREAD

**Discussion by andygomez3dart** [google/model-viewer#2295] [Q&A]
On mobile version camera controls to orbit in y axis doesn't work. it always scrolls down by default.

Anyone knows how to enable  orbit/rotation in y axis and not only in X axis?
I googled a lot and couldn't find the answer so far.

I used the code for panning the 3d model
Could this be the problem?

In desktop version works fine and I can orbit in X and y axis as wells pan the 3d model.

I hope somebody can help.
Many thanks.

Andy.

**Comment by elalish** [comment]
Yes, that is our default behavior so that the page is still scrollable. If you drag to the side, you can then orbit all ways. If you want to swallow the scroll events instead, you'll want to set `touch-action="none"`: https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-touch-action

**Reply by elalish** [reply]
Also, we usually call the up-down rotation the X-axis and the side-to-side rotation the Y-axis.

**Reply by andygomez3dart** [reply]
Thanks very much for your help. I will give it a try and see how it goes

**Reply by andygomez3dart** [reply]
> Thanks very much for your help. I will give it a try and see how it goes

That worked like a charm, exactly what I was looking for!
